main/options/options.cc

```diff
@@ -320,6 +320,11 @@ buildOptions(const vector<pipeline::semantic_extension::SemanticExtensionProvide
         "End users should prefer to use `--enable-all-beta-lsp-features`, instead.)");
     options.add_options("advanced")("enable-all-beta-lsp-features",
                                     "Enable (expected-to-be-non-crashy) early-access LSP features.");
+    options.add_options("advanced")(
+        "lsp-max-errors-reported",
+        "Limits the maximum number of simultaneous errors that LSP reports to the editor. Can prevent "
```

Comment:
what do you mean by "simultaneous errors"?

Reply:
The number of errors that the client reports at one point.

I guess removing 'simultaneous' here is still clear.